testfiles/HeatPumpWaterToAirEquationFit_WaterHeatingDesuperheater_StratifiedTank.idf

```diff
@@ -0,0 +1,3988 @@
+
+  Zone,
+    SPACE4-1,                !- Name
+    0,                       !- Direction of Relative North {deg}
+    0,                       !- X Origin {m}
+    0,                       !- Y Origin {m}
+    0,                       !- Z Origin {m}
+    1,                       !- Type
+    1,                       !- Multiplier
+    2.438400269,             !- Ceiling Height {m}
+    103.311355591;           !- Volume {m3}
+
+  ZoneInfiltration:DesignFlowRate,
+    SPACE4-1 Infiltration,   !- Name
+    SPACE4-1,                !- Zone or ZoneList Name
+    INFIL-SCH,               !- Schedule Name
+    flow/zone,               !- Design Flow Rate Calculation Method
+    .0717,                   !- Design Flow Rate {m3/s}
+    ,                        !- Flow per Zone Floor Area {m3/s-m2}
+    ,                        !- Flow per Exterior Surface Area {m3/s-m2}
+    ,                        !- Air Changes per Hour {1/hr}
+    0,                       !- Constant Term Coefficient
+    0,                       !- Temperature Term Coefficient
+    0.2237,                  !- Velocity Term Coefficient
+    0;                       !- Velocity Squared Term Coefficient
+
+  People,
+    SPACE4-1 People,         !- Name
+    SPACE4-1,                !- Zone or ZoneList Name
+    OCCUPY-1,                !- Number of People Schedule Name
+    people,                  !- Number of People Calculation Method
+    5,                       !- Number of People
+    ,                        !- People per Zone Floor Area {person/m2}
+    ,                        !- Zone Floor Area per Person {m2/person}
+    0.3,                     !- Fraction Radiant
+    ,                        !- Sensible Heat Fraction
+    ActSchd;                 !- Activity Level Schedule Name
+
+  Lights,
+    SPACE4-1 Lights,         !- Name
+    SPACE4-1,                !- Zone or ZoneList Name
+    LIGHTS-1,                !- Schedule Name
+    LightingLevel,           !- Design Level Calculation Method
+    684,                     !- Lighting Level {W}
+    ,                        !- Watts per Zone Floor Area {W/m2}
+    ,                        !- Watts per Person {W/person}
+    0.2,                     !- Return Air Fraction
+    0.59,                    !- Fraction Radiant
+    0.2,                     !- Fraction Visible
+    0,                       !- Fraction Replaceable
+    GeneralLights;           !- End-Use Subcategory
+
+  ElectricEquipment,
+    SPACE4-1 Elec Equipment, !- Name
+    SPACE4-1,                !- Zone or ZoneList Name
+    EQUIP-1,                 !- Schedule Name
+    EquipmentLevel,          !- Design Level Calculation Method
+    456,                     !- Design Level {W}
+    ,                        !- Watts per Zone Floor Area {W/m2}
+    ,                        !- Watts per Person {W/person}
+    0,                       !- Fraction Latent
+    0.3,                     !- Fraction Radiant
+    0;                       !- Fracti
```

Comment:
This needs to be fixed:
```
   ** Warning ** ProcessScheduleInput: DecodeHHMMField, Invalid "until" field value is not a multiple of the minutes for each timestep: UNTIL: 13:10
   **   ~~~   ** Other errors may result. Occurred in Day Schedule=HOT WATER DEMAND SCHEDULE_dy_1
```
There are other warnings with this idf which may or may not be easy to fix:
```
   ** Warning ** Plant Loop: WATER SOURCE HEAT PUMP CONDENSER LOOP Demand Side is storing excess heat the majority of the time.
   ** Warning ** Plant Loop: WATER SOURCE HEAT PUMP CONDENSER LOOP Supply Side is storing excess heat the majority of the time.

   *************  ** Warning ** Plant loop falling below lower temperature limit, PlantLoop="WATER SOURCE HEAT PUMP CONDENSER LOOP"
   *************  **   ~~~   **   This error occurred 1 total times;
   *************  **   ~~~   **   during Warmup 1 times;
   *************  **   ~~~   **   during Sizing 0 times.
   *************  **   ~~~   **   Min=4.899975
```

Reply:
I locally fixed the first one, for the second one, I noticed the ``HeatPumpWaterToAirEquiationFit.idf`` test file (it's one of files where I referred to generate this desuperheater test file) generated the same warning. It might be caused elsewhere out of this PR scope?

Reply:
Thanks for fixing the schedule error. Yes, both have plant issues, won't hold this up here for that.